Examples/Scripts/Digitization/error_parameterisation.py

```diff
@@ -0,0 +1,280 @@
+                             break_rms_change = 0.05,
+                             break_cluster_size = 5, 
+                             view_colors =  [ 'deepskyblue', 'gold' ], 
+                             view_rms_range = 5) :
+
+
+    # Create a figure directory
+    output_dir.mkdir(parents=True, exist_ok=True)
+    output_html_dir = output_dir / "html"
+    output_html_dir.mkdir(parents=True, exist_ok=True)
+    output_fig_dir = output_html_dir / "plots"
+    output_fig_dir.mkdir(parents=True, exist_ok=True)
+
+    volume_links = ''
+
+    logging.info(f'Hit error parameterisation for {len(volumes)} volumes')
+
+
+    var_dict = {}
+    header_dict = {}
+    header_dict['format-version'] = 0
+    header_dict['value-identifier'] = 'hit-error-parametrisation'
+    var_dict['acts-geometry-hierarchy-map'] = header_dict
+
+    var_entries = []
+
+    # loop over the volumes
+    for iv, v_id_n in enumerate(volumes) :
+
+        v_id, v_name = v_id_n
+
+        logging.info(f'Processing volume {v_name} with ID: {v_id}')
+
+        # previos and next volume
+        prev_id = volumes[iv-1] if iv > 0 else volumes[-1]
+        next_id = volumes[iv+1] if iv < len(volumes)-1 else volumes[0]
+
+        # Get the volume
+        v_id_str = 'vol'+str(v_id)
+        vol = rfile[v_id_str].arrays(library='pd')
+       
+        # RMS matrix
+        max_size_0 = 1
+        max_size_1 = 1
+
+
+        # We should be able to get this from the volume
+        local_values = [ ]
+        if 'clus_size_loc0' in vol.columns and vol['clus_size_loc0'].any() :
+            logging.info(f' - local 0 coorindate found')
+            local_values.append(0)
+        if 'clus_size_loc1' in vol.columns and vol['clus_size_loc1'].any() :
+            local_values.append(1)
+            logging.info(f' - local 1 coorindate found')
+       
+    
+        var_matrix = np.zeros((2, break_cluster_size))
+        var_entry = { 'volume' : v_id}
+        var_data = []
+
+        # write html content
+        plots = []
+        # Loop over the local variables
+        for l in local_values :
+            # Local var_data
+            rms_local_values = { 'index' : l }
+            rms_local_data = []
+            # The plots per column
+            lplots = []
+            # Overview plot
+            plt.hist(vol['clus_size_loc'+str(l)], 
+                    bins = range(1, max(vol['clus_size_loc'+str(l)])+3),
+                    histtype='step', fill=True, color=view_colors[l])
+            plt.xlabel('Cluster size local '+str(l))
+            plt.ylabel('Entries')
+            # Create the svg path
+            svg_path = output_fig_dir / f'{v_id_str}_clus_size_loc{l}.svg'
+            plt.savefig(svg_path)
+            lplots.append(svg_path)
+            plt.clf()
+            # Resolution plot, break 
+            max_clus_size = max(vol['clus_size_loc'+str(l)])+1
+            if max_clus_size > break_cluster_size :
+                max_clus_size = break_cluster_size
```

Comment:
The `var_matrix` is not updated to reflect this?

Reply:
not necessary, max will be set to break, the array is already allocated to break